src/Microsoft.ComponentDetection.Detectors/spdx/Spdx22ComponentDetector.cs

```diff
@@ -91,7 +91,7 @@ private SpdxComponent ConvertJObjectToSbomComponent(ProcessRequest processReques
             var name = document["name"]?.ToString();
             var spdxVersion = document["spdxVersion"]?.ToString();
 
-            if (rootElements.Length > 1)
+            if (rootElements != null && rootElements.Length > 1)
```

Comment:
I'm not sure if this is valid syntax, but you might be able to combine this into a single expression.

```suggestion
            if (rootElements?.Length > 1)
```